drivers/wasapi/audio_driver_wasapi.h

```diff
@@ -38,15 +38,16 @@
 #include "servers/audio_server.h"
 
 #include <audioclient.h>
+#include <atlbase.h>
```

Comment:
Is it available for MinGW? I do not see it in my installation (MinGW 9.0.0).

Reply:
Unfortunately no, ATL is a part of Visual C++.

Reply:
If it's MSVC only, we can't use it in Godot. We want cross-compilation support, and use MinGW for all official releases.

Reply:
Okay, no problem. 🙂